alvr/server/src/lib.rs

```diff
@@ -63,7 +49,7 @@ use std::{
 use sysinfo::{ProcessRefreshKind, RefreshKind};
 use tokio::{runtime::Runtime, sync::broadcast};
 
-static FILESYSTEM_LAYOUT: Lazy<Layout> = Lazy::new(|| {
+static FILESYSTEM_LAYOUT: Lazy<afs::Layout> = Lazy::new(|| {
```

Comment:
is this even needed here anymore? It's openvr specific and also present in the openvr code

Reply:
Hm. I guess this will have to be solved. But if it's ok with you we can deal with this later.

Reply:
sure, later is fine